install/sample/proxy.yaml

```diff
@@ -0,0 +1,157 @@
+apiVersion: v1
```

Comment:
How are these resources used?

Reply:
@nirvanagit , these resources are used in the integration test